opm/output/eclipse/AggregateUDQData.cpp

```diff
@@ -572,356 +749,379 @@ namespace {
         allocate(const std::vector<int>& udqDims)
         {
             using WV = Opm::RestartIO::Helpers::WindowedArray<int>;
-            int nwin = std::max(udqDims[7], 1);
+
+            const int nwin = std::max(udqDims[7], 1);
```

Comment:
I find this expression kinda confusing, is udqDims[7] >= 0 or not ?  if it is i'd find a ternary more expressive, if not the return value below seems suspicious.

Reply:
The original code was somewhat questionable and pushed the [check for non-zero size]() to `RestartIO.cpp`.  I've pushed an update that makes the UDA-related arrays `optional<>` instead which means we get the same kind of call site logic for those as we do for the `DUDx` arrays.